src/devices/video/ppu2c0x.cpp

```diff
@@ -263,10 +263,7 @@ void ppu2c0x_device::device_start()
 {
 	start_nopalram();
 
-	m_palette_ram.resize(0x20);
-
-	for (int i = 0; i < 0x20; i++)
-		m_palette_ram[i] = 0x00;
+	m_palette_ram.resize(0x20, 0x00);
```

Comment:
This may not do what you expect if the size wasn’t zero before the call to resize.  You need to empty it then resize it if you want to ensure the fill-on-resize actually fills it.

Reply:
It only gets resized once to its fixed size in device_start here (and likewise in each of the derivative VT PPU clone device_start calls).